pkg/registry/rbac/validation/rule.go

```diff
@@ -176,11 +177,14 @@ func (d *roleBindingDescriber) String() string {
 }
 
 func (r *DefaultRuleResolver) VisitRulesFor(user user.Info, namespace string, visitor func(source fmt.Stringer, rule *rbacv1.PolicyRule, err error) bool) {
+
+	userTenant := user.GetTenant()
+
 	if clusterRoleBindings, err := r.clusterRoleBindingLister.ListClusterRoleBindings(); err != nil {
```

Comment:
As we discussed before, the ClusterRole and ClusterRoleBinding will be changed to tenant-scope. I will send out a PR on that. And you can make changes to get the rolebindings for the specific tenant only.

Reply:
The change is now ready at https://github.com/futurewei-cloud/arktos/pull/117

Reply:
taken